crates/net/network/src/session/active.rs

```diff
@@ -253,7 +253,11 @@ impl<N: NetworkPrimitives> ActiveSession<N> {
                 on_response!(resp, GetNodeData)
             }
             EthMessage::GetReceipts(req) => {
-                on_request!(req, Receipts, GetReceipts)
+                if self.conn.version().is_eth69() {
```

Comment:
let's do >= EthVersion::eth69 here because we assume this will be the case for new versions as well